docs/reference/intro.asciidoc

```diff
@@ -146,90 +170,197 @@ A mapping defines the <<mapping-types,data type>> for each field, how the field
+
+General content is data that does not have a timestamp.
+This could be data like vector embeddings, website content, product catalogs, or more.
+For general content, you have the following options for adding data to {es} indices:
+
+* <<docs,API>>: Use the {es} <<docs,Document APIs>> to index documents directly, using the Dev Tools {kibana-ref}/console-kibana.html[Console], cURL.
+** You can use https://www.elastic.co/guide/en/elasticsearch/client/index.html[{es} programming language clients] to index documents in your programming language of choice. For Python devs, check out the `elasticsearch-labs` repo for various https://github.com/elastic/elasticsearch-labs/tree/main/notebooks/search/python-examples[example notebooks]. 
+* {kibana-ref}/connect-to-elasticsearch.html#upload-data-kibana[File upload]: Use the {kib} file uploader to upload and index CSV, JSON, and log files.
+* {kibana-ref}/connect-to-elasticsearch.html#_add_sample_data[Sample data]: Load sample data sets into your {es} cluster using {kib}.
+* https://github.com/elastic/crawler[Web crawler]: Extract and index web page content into {es} documents.
+* {enterprise-search-ref}/connectors.html[Connectors]: Sync data from various third-party data sources to create searchable, read-only replicas in {es}.
+
+[discrete]
+[[es-ingestion-overview-timestamped]]
+==== Timestamped data
+
+For timestamped data, you have the following options for adding data to {es} data streams:
+
+* {fleet-guide}/fleet-overview.html[Elastic Agent and Fleet]: The preferred way to index timestamped data. Each Elastic Agent based integration includes default ingestion rules, dashboards, and visualizations to start analyzing your data right away.
+You can use the Fleet UI in {kib} to centrally manage Elastic Agents and their policies.
+* {beats-ref}/beats-reference.html[Beats]: If your data source isn't supported by Elastic Agent, use Beats to collect and ship data. You install a separate Beat for each type of data to collect.
+* {logstash-ref}/introduction.html[Logstash]: Logstash is an open source data collection engine with real-time pipelining capabilities that supports a wide variety of data sources. You might use this option because neither Elastic Agent nor Beats supports your data source. You can also use Logstash to persist incoming data, or if you need to send the data to multiple destinations. 
+* {cloud}/ec-ingest-guides.html[Language clients]: The linked tutorials demonstrate how to use {es} programming language clients to ingest data from an application. (In these examples, {es} is running on Elastic Cloud, but the same principles apply to any {es} deployment.)
+
+[TIP]
+====
+If you're interested in data ingestion pipelines for timestamped data, use the decision tree in the {cloud}/ec-cloud-ingest-data.html#ec-data-ingest-pipeline[Elastic Cloud docs] to understand your options.
+====
+
 // New html page
 [[search-analyze]]
-=== Search and analyze
+=== Search and analyze data
```

Comment:
I'm hesitant to start refactoring this again because this PR is out of control

I totally agree with the flow, but honestly I'm not sure how well any of this flows. 

> I think this page is better

This is the acceptance criteria

Reply:
All of this stuff will probably be revisited one way or another quite soon so I don't think we need to polish this excessively immediately

Reply:
lgtm